Bugzilla/Quantum/Plugin/SizeLimit.pm

```diff
@@ -0,0 +1,77 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# This Source Code Form is "Incompatible With Secondary Licenses", as
+# defined by the Mozilla Public License, v. 2.0.
+
+package Bugzilla::Quantum::Plugin::SizeLimit;
+use 5.10.1;
+use Mojo::Base 'Mojolicious::Plugin';
+use Mojo::JSON qw(decode_json);
+use Bugzilla::Logging;
+
+use constant MIN_SIZE_LIMIT        => 750_000;
+use constant HAVE_LINUX_SMAPS_TINY => eval { require Linux::Smaps::Tiny };
+use constant HAVE_BSD_RESOURCE     => eval { require BSD::Resources };
```

Comment:
BSD::Resource

Reply:
Fixed, also listed BSD::Resource as part of the linux_smaps feature because it's not actually a dep right now -- it just happens to exist on centos 6!